### DISCUSSION TITLE
sitemap.xml not generating

### DISCUSSION THREAD

**Discussion by steviebuilds** [facebook/docusaurus#4072] [Q&A] [answered]
Not sure if this is an issue or just me so I'll start here.

I can't seem to find the sitemap.xml file when building my d2 alpha-70 project. I noticed [the docs](https://v2.docusaurus.io/docs/api/plugins/@docusaurus/plugin-sitemap/) mention in the admonition: if you've used the preset classic theme you don't need to install the sitemap plugin to the project. Either way, with or without, I can't seem to generate it. 

I'm hoping to get a hold of it to upload it to fix up my site's google search indexing.

Any help would be appreciated.

**Comment by slorber** [comment, marked as answer]
Hi,

On the docusaurus site itself, we don't have any additional sitemaps config and it's generated: https://v2.docusaurus.io/sitemap.xml

It's nearly impossible to help you without the live URL of your site and if possible source code access.

**Reply by steviebuilds** [reply]
Ah yep, I've found it... not sure where I was looking before but obviously not in the right place 🤦‍♂️. Thanks!
